### ISSUE TITLE
Built-in regex's do not support brace-expressions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ThomasDickey/original-mawk#25]
```
On Sat, Aug 09, 2014 at 11:34:09PM -0700, Leif LeBaron wrote:
> I didn't see any mention in the bug tracker

thanks for the reminder (I thought I'd created issues for "everything" 
commented on by others...).

in my own to-do list, I have this note:

        120802 mawk says it uses "extended regular" expressions, but doesn't
        appear to honor brace-expressions (verify).
        121027 that appears here in gawk: --re-interval

> so here's a request for more regex quantifiers
> you know:
>
>       {n}
>       {,n}
>       {m,}
>       {m,n}



```

Original issue reported on code.google.com by `dic...@his.com` on 10 Aug 2014 at 9:10

**Comment by ThomasDickey** [comment]
current mawk supports braces, but does not handle grouping with braces

**Comment by ThomasDickey** [comment]
fixed in [20230730](https://invisible-island.net/mawk/CHANGES.html#t20230730) (the remaining work to resolve the bug that I had in mind was actually not much).
